tests/__snapshots__/css-import.test.ts.snap

```diff
@@ -462,11 +462,6 @@ input::-moz-placeholder, textarea::-moz-placeholder {
   color: #a1a1aa;
 }
 
-input:-ms-input-placeholder, textarea:-ms-input-placeholder {
-  opacity: 1;
-  color: #a1a1aa;
-}
```

Comment:
@NullVoxPopuli do you happen to know why the prefixes would be stripped out here?  All I did was add a shadow 😭

Reply:
my hunch would be autoprefixer / browserslist changes (which involve floating deps).

the `-ms` prefix was for IE, and idk if anyone uses IE anymore

Reply:
Niceeeee! I figured it was likely something with autoprefixer.  I think this is mostly safe.  There's some `-webkit` stuff in here too, but everything _looks_ safe to me.